ui/app/AppLayouts/Wallet/popups/AddAccountModal.qml

```diff
@@ -272,21 +265,38 @@ StatusModal {
         StatusButton {
             id: nextButton
 
-            text: loading ? qsTr("Loading...") : qsTr("Add account")
+            text: {
+                if (d.authenticationNeeded) {
+                    return qsTr("Authenticate")
+                }
+                if (loading) {
+                    return qsTr("Loading...")
+                }
+                return qsTr("Add account")
+            }
+
 
             enabled: {
+                if (d.authenticationNeeded) {
+                    return true
+                }
                 if (loading) {
                     return false
                 }
+                return accountNameInput.text !== "" && advancedSelection.isValid
+            }
 
-                return (advancedSelection.expandableItem.addAccountType === SelectGeneratedAccount.AddAccountType.WatchOnly || d.isPasswordCorrect)
-                        && accountNameInput.text !== "" && advancedSelection.isValid
+            icon.name: {
```

Comment:
`d.authenticationNeeded ? d.addAccountIcon : ""` looks like we are not using many of js in QML ;)